pilot/pkg/model/service.go

```diff
@@ -88,6 +88,12 @@ type Service struct {
 
 	// CreationTime records the time this service was created, if available.
 	CreationTime time.Time `json:"creationTime,omitempty"`
+
+	// Namespace groups Services - it corresponds to a k8s namespace, or equivalent
+	// construct for other registries. For example a label may be used in registries
+	// not having a native namespace concept. Used for mixer attributes and to identify
+	// the source of the service.
+	Namespace string
```

Comment:
This is not good. Pretty certain that people will start mis using it in networking code and construct FQDNs out of this value.  Rather than add this, why not have a generic metadata field that has all this info? that metadata can be populated from anything. Mixer can then stick in a variety of info here. Otherwise, we will be extending the service struct forever with more and more fields like this..